docs/configuration.rst

```diff
@@ -482,184 +424,371 @@ The following environment variables for the tracer are supported:
+
+   DD_COMPILE_DEBUG:
+     type: Boolean
+     default: False
+     description: Compile Cython extensions in RelWithDebInfo mode (with debug info, but no debug code or asserts)
+
+   DD_INSTRUMENTATION_TELEMETRY_ENABLED:
+     type: Boolean
+     default: True
+     
+     description: |
+         Enables sending :ref:`telemetry <Instrumentation Telemetry>` events to the agent.
+
+   DD_RUNTIME_METRICS_ENABLED:
+     type: Boolean
+     default: False
+     
+     description: |
+         When used with ``ddtrace-run`` this configuration enables sending runtime metrics to Datadog.
+         These metrics track the memory management and concurrency of the python runtime. 
+         Refer to the following `docs <https://docs.datadoghq.com/tracing/metrics/runtime_metrics/python/>` _ for more information.
+
+   DD_SUBPROCESS_SENSITIVE_WILDCARDS:
+     type: String
+     
+     description: |
+         Add more possible matches to the internal list of subprocess execution argument scrubbing. Must be a comma-separated list and
+         each item can take `fnmatch` style wildcards, for example: ``*ssn*,*personalid*,*idcard*,*creditcard*``.
+
+   DD_USER_MODEL_EMAIL_FIELD:
       type: String
-      default: "safe"
+      default: ""
+      
       description: |
-         Sets the mode for the automated user login events tracking feature which sets some traces on each user login event. The
-         supported modes are ``safe`` which will only store the user id or primary key, ``extended`` which will also store
-         the username, email and full name and ``disabled``. Note that this feature requires ``DD_APPSEC_ENABLED`` to be
-         set to ``true`` to work.
+         Field to be used to read the user email when using a custom ``User`` model for the automatic login events. This field will take precedence over automatic inference.
+      
       version_added:
-         v1.17.0: Added support to the Django integration. No other integrations support this configuration.
+         v1.15.0:
 
    DD_USER_MODEL_LOGIN_FIELD:
       type: String
       default: ""
+      
       description: |
          Field to be used to read the user login when using a custom ``User`` model for the automatic login events. This field will take precedence over automatic inference.
          Please note that, if set, this field will be used to retrieve the user login even if ``DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING`` is set to ``safe`` and,
          in some cases, the selected field could hold potentially private information.
-      version_added:
-         v1.15.0:
-
-   DD_USER_MODEL_EMAIL_FIELD:
-      type: String
-      default: ""
-      description: |
-         Field to be used to read the user email when using a custom ``User`` model for the automatic login events. This field will take precedence over automatic inference.
+      
       version_added:
          v1.15.0:
 
    DD_USER_MODEL_NAME_FIELD:
```

Comment:
Quick question, how have you categorized them?

Reply:
I generally categorized by product, which is probably still too high-level. I'm trying to get some feedback (mainly from API SDK and ASM) about possibly bucketing these at a lower level/feature level instead (I kinda did this for sampling already).

Reply:
@erikayasuda -- very nice! If you're looking for some lower level categories, you could see how we recently reorganized these options for all languages on docs.datadoghq.com: https://docs.datadoghq.com/tracing/trace_collection/library_config/.

These are roughly the categories we tried to use:

- Unified service tagging
- Traces
- Agent
- ASM
- Databases
- Database monitoring
- Errors
- Logs
- OpenTelemetry
- Profiling
- Trace context propagation